example/advection/advection_package.cpp

```diff
@@ -39,6 +39,8 @@ std::shared_ptr<StateDescriptor> Initialize(ParameterInput *pin) {
   pkg->AddParam<>("vx", vx);
   Real vy = pin->GetOrAddReal("Advection", "vy", 1.0);
   pkg->AddParam<>("vy", vy);
+  Real vz = pin->GetOrAddReal("Advection", "vz", 0.5);
```

Comment:
@pgrete just curious, why default to 0.5?  I used 1.0 originally for the other directions to make it transit the unit box exactly once per unit time.

Reply:
I wanted to have sth non-trivial, but I also have no objections changing the default.